pkgs/misc/uboot/default.nix

```diff
@@ -194,6 +194,17 @@ in {
     filesToInstall = ["u-boot-sunxi-with-spl.bin"];
   };
 
+  ubootEspressobin = buildUBoot rec {
+    defconfig = "mvebu_espressobin-88f3720_defconfig";
+    extraMeta.platforms = ["aarch64-linux"];
+    filesToInstall = ["u-boot.bin"];
+    src = fetchurl {
+      url = "ftp://ftp.denx.de/pub/u-boot/u-boot-${version}.tar.bz2";
+      sha256 = "03wm651ix783s4idj223b0nm3r6jrdnrxs1ncs8s128g72nknhk9";
+    };
+    version = "2023.01";
```

Comment:
Like @lopsided98 noticed, we should be updating U-Boot first.

We should probably block this PR until our U-Boot build is updated for 2023.01. And then remove this.

Reply:
Alright.  The u-boot code does work on older revisions, just that I contributed a patch that fixes overwriting environment variables that only made it into 2023.01.  Would you like me to add a revision to remove the version?